tests/State/HubTest.php

```diff
@@ -292,4 +293,13 @@ public function testCaptureEvent(): void
 
         $this->assertEquals('2b867534eead412cbdb882fd5d441690', $hub->captureEvent(['message' => 'test']));
     }
+
+    protected function getScope(HubInterface $hub): Scope
+    {
+        $class = new \ReflectionClass($hub);
```

Comment:
We can use `ReflectionMethod` directly instead of passing for the class metadata